l3kernel/doc/source3body.tex

```diff
@@ -335,6 +335,14 @@ \section{Naming functions and variables}
     lists (with access at both ends) and stacks.
 \end{description}
 
+\subsection{Behavior of \texttt{c}-type expansion for undefined arguments}
```

Comment:
the arguments aren't undefined the produced token is, but I have no idea how to say that

Reply:
```suggestion
\subsection{Behavior of \texttt{c}-type expansion for undefined control sequence names}
```

Reply:
yes, better

Reply:
Agreed!

Reply:
I was avoiding 'control sequence' as `expl3` talks about functions and variables - but I can see that perhaps we have no choice here